src/test/app-tenant.yaml

```diff
@@ -26,7 +26,6 @@ spec:
       - service: test-tenant-app-cidr
         selector:
           app: test-tenant-app
-        remoteCidr: "192.168.0.0/16"
```

Comment:
This [was added ](https://github.com/defenseunicorns/uds-core/pull/719/files#diff-c94db7cc6e40995709784aae6c1a7f05ae2ed175ca1c0eb2553de353ddbd5d5fL25-L26)for testing that the remoteCidr generation wouldn't break anything. If you're going to remove this then the whole netpol should be removed since that was its only usecase.